src/scrabble.js

```diff
@@ -1,5 +1,53 @@
-function scrabble() {
+let score = 0
 
+function findValues(word) {
+    let value1 = 0
+    // console.log(`what is i: ${i}`)
+    let comparisonLetter = word.charAt(i)
```

Comment:
I like that this index links to the for loop thats at the bottom instead of making a new loop for it